spec/machine/ARM/MachineOps.thy

```diff
@@ -143,25 +143,29 @@ definition
 where
   "MAX_PERIOD_US \<equiv> 60 * 60 * 1000 * 1000"
 
-axiomatization
-  us_to_ticks :: "ticks \<Rightarrow> ticks"
-where
-  us_to_ticks_mono[intro!]: "mono us_to_ticks"
+definition "\<mu>s_in_ms = 1000"
+
+consts' factor1 :: ticks
+consts' factor2 :: ticks
```

Comment:
We should give these better names. I was just using them to explain the form the term should take. Maybe something like `ticks_per_timer` for `factor1` and `timer_unit` for `factor2`.

Reply:
better `ticks_per_timer_unit` than `ticks_per_timer`

Reply:
Thanks, those sound good